graphql/resolvers/User.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  name (user) {
+    return [user.firstName, user.lastName].join(' ')
+  },
```

Comment:
user.name is required in many places, thats why we have lib/factories/createUser. I would use this here.